README.md

```diff
@@ -1,12 +1,17 @@
 # **Textsnippet** #
-| Textsnippet lets you generate random texts.
-| and there is an option for speach.
+> Textsnippet lets you generate random texts.
+<br/>
+
+> and there is an option for speech.
 
 <br/>
 
 # **Getting Started** #
 
 * Download or clone the repository to the computer .
+![screenshot](/readme_imgs/4.png)
+![screenshot](/readme_imgs/5.png)
+![screenshot](/readme_imgs/6.png)
```

Comment:
No need screenshots here. whoever uses git, they know how to clone it, so , change it to:
* Clone repository
```bash
git clone <url here>
```